spec/models/team_spec.rb

```diff
@@ -12,7 +12,34 @@
 
 RSpec.describe Team, type: :model do
   it "is valid when produced by a factory" do
-    team = FactoryBot.build(:team)
+    team = FactoryBot.create :team
     expect(team).to be_valid
   end
+
+  it "should have and belong to team owners" do
+    team = FactoryBot.create :team
+    # After building a team via FactoryBot an owner is created using the factory for users and assigned to the team
+    expect(team.owners).to have(1).items
+  end
+
+  it "should have and belong to team members" do
+    # After building a team via FactoryBot a member is created using the factory for users and assigned to the team
+    team = FactoryBot.create :team
+    expect(team.members).to have(1).items
+  end
+
+  it "should be able to have multiple team owners" do
+    team = FactoryBot.create :team, :with_multiple_owners
+    expect(team.team_owners).to have(2).items
```

Comment:
I think it's not really clear how many team owners a freshly created team has since the trait `:with_multiple_owners` is somewhat ambiguous...
I'd either give the trait a more descriptive name (e.g. `:with_two_owners`) or change the trait to accept the number of owners as parameter.

Reply:
Implemented the first approach to use more explicit trait names.